pyproject.toml

```diff
@@ -137,6 +138,9 @@ requires-python = '>=3.9'
 'process.workflow.workchain' = 'aiida.orm.nodes.process.workflow.workchain:WorkChainNode'
 'process.workflow.workfunction' = 'aiida.orm.nodes.process.workflow.workfunction:WorkFunctionNode'
 
+[project.entry-points.'aiida.orm']
+'core.code.abstract' = 'aiida.orm.nodes.data.code.abstract:AbstractCode'
```

Comment:
I think this entry point should be moved to `[project.entry-points.'aiida.data']` section where the other code entry points are defined.